test/model/compose_test.dart

```diff
@@ -314,6 +314,28 @@ hello
         '#narrow/dm/1,2-dm/near/12345',
         '#narrow/pm-with/1,2-pm/near/12345');
     });
+
+    test('normalize links to always include a "/" after hostname', () {
```

Comment:
The code this is testing is in `internal_link.dart`, not `compose.dart`, so let's place the test to match. We can do that with a prep commit moving this `narrowLink` test group to the top of `internal_link_test.dart`; really that should have happened as part of 2f944a0125f67aed2df47a02e1f254ac76710179.